python/chemiscope/explore.py

```diff
@@ -197,6 +198,121 @@ def soap_pca_featurize(frames, environments=None):
     return pca.fit_transform(feats)
 
 
+def metatensor_featurizer(model, extensions_directory="extensions/", device=None):
```

Comment:
I would add `check_consistency` here as well, to follow the API of all other metatensor integrations